generateapis.sh

```diff
@@ -38,26 +39,46 @@ else
   declare -r GOOGLEAPIS="$PWD/googleapis"
 fi
 
-# TODO: Work out whether we need synthtool handling etc.
-declare -r GOOGLEAPIS_DISCOVERY="$PWD/googleapis-discovery"
+# Handle either a locally-fetched googleapis-discovery, or
+# one specified in a preconfig file.
+if [[ "$SYNTHTOOL_PRECONFIG_FILE" != "" ]]
+then
+  declare -r GOOGLEAPIS=$(python - <<END
```

Comment:
Should this one be GOOGLEAPIS_DISCOVERY? Not sure though.

Reply:
Yes it should! Thanks and we'll spotted